content/03.ml-concepts.md

```diff
@@ -1,2 +1,14 @@
 ## Rule 1: Concepts that apply to machine learning also apply to deep learning
+Deep learning is a distinct subfield of machine learning, but it is still a subfield.
```

Comment:
If we don't mention it in a prior introduction, I suggest that we also briefly mention what deep learning "is." Maybe sth like

"While deep learning is a distinct subfield of machine learning, that is particularly focused on deep artificial neural networks, it is still a subfield."

Reply:
The current first line of the intro is "Deep learning (DL), a subfield of machine learning (ML) that focuses on deep artificial neural networks, is exploding in popularity and is increasingly being used for biological data analysis"

Reply:
Oh, I forgot that I already suggested it for the intro in https://github.com/Benjamin-Lee/deep-rules/pull/83. I knew it sounded familiar :D. Never mind this suggestion here then; would be redundant